attachment_unindex_content/README.rst

```diff
@@ -0,0 +1,75 @@
+==========================
```

Comment:
Better using readme folder like template:
 - https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
I  just saw that it was created fine!
So Forget this comment